cap/01-categorie.tex

```diff
@@ -1,9 +1,162 @@
+ \end{itemize}
+ Ad ogni oggetto $x$, l'identità è l'unico morfismo $x\to x$ dato dal fatto che la relazione è riflessiva ($x\sim x$). 
+ La composizione è data dalla transitività: se abbiamo morfismi $x\to y$ e $y\to z$ significa, in particolare, che $x\sim y$ e $y\sim z$. Per transitività, $x\sim z$, e quindi c'è un unico morfismo $x\to z$, che possiamo prendere come composizione. 
+ Gli assiomi di unitalità e associatività sono automaticamente soddisfatti: per esempio, dati $w\sim x\sim y\sim z$, per transitività abbiamo che $w\sim z$, e questo dà un \emph{unico} morfismo $w\to z$, non importa se lo otteniamo componendo prima $w\sim x$ e $x\sim y$ o prima $x\sim y$ e $y\sim z$ -- il risultato è lo stesso per unicità. 
+\end{example}
+
+Nell'esempio sopra vediamo che \emph{gli oggetti non sono insiemi con strutture ulteriori}. 
+Notiamo anche che la proprietà di \emph{simmetria} delle relazioni di equivalenza non serve per avere una categoria. Consideriamo i seguenti esempi.
+
+\begin{examples}
+ Una relazione d'ordine $(X,\le)$ si può vedere come una categoria, ancora una volta con un unico morfismo $x\to y$ se e solo se $x\le y$. Come nel caso delle relazioni di equivalenza, le identità sono date dalla proprietà riflessiva e la composizione dalla proprietà transitiva. 
+ 
+ Più in generale, un \emph{preordine} è una relazione riflessiva e transitiva, ma non necessariamente simmetrica o antisimmetrica. Ogni preordine si può vedere come una categoria.
+\end{examples}
+
+
+\begin{example}
+ Un monoide si può vedere come una categoria con un solo oggetto -- più precisamente, una categoria \emph{localmente piccola} con un solo oggetto. 
+ Vediamo come. Dato un monoide $M$, definiamo la seguente categoria, che indichiamo con $\ctB M$.\footnote{A volte la categoria definita in questo modo si indica con $\ctB M$, a volte semplicemente con $M$.}
+ \begin{itemize}
+  \item La categoria $\ctB M$ ha un unico oggetto, che indichiamo con un punto, $\bullet$;
+  \item La categoria $\ctB M$ ha un morfismo $m:\bullet\to\bullet$ per ogni elemento $m\in M$;
+  \item L'identità dell'unico oggetto $m$ e il morfismo definito dall'elemento neutro $e\in M$;
+  \item La composizione di morfismi è data dal prodotto in $M$.
+ \end{itemize}
+ In un monoide, a differenza di una categoria generica, possiamo sempre comporre due morfismi $m$ e $n$: questo è garantito dal fatto che siccome c'è un solo oggetto, il dominio e il codominio di $m$ ed $n$ sono necessariamente uguali. 
+ 
+ Viceversa, data una categoria localmente piccola con un solo oggetto (indichiamolo ancora con questo simbolo, $\bullet$), l'insieme di morfismi $\bullet\to\bullet$ ha una struttura di monoide con l'elemento neutro dato dall'identità, e il prodotto dato dalla composizione.
+ 
+ L'unitalità e l'associatività del monoide e della categoria si corrispondono.
+\end{example}
+
+In particolare, ogni gruppo si può vedere come una categoria con un solo oggetto. 
```

Comment:
"...e i gruppi sono precisamente le categorie dove..."

Reply:
Faccio il puntiglioso, ma è davvero giusto dire che i gruppi "sono precisamente le categorie dove…"? Quello che (immagino) vogliamo dire è che i dati e le proprietà che definiscono un gruppo sono gli stessi che definiscono una categoria dove etc… È la differenza tra identità e isomorfismo.

Potremmo dire che: "i gruppi sono (canonicamente in bigezione con) le categorie dove etc…" senza poi necessariamente ripetere la parte sulla bigezione ogni volta.

Reply:
Sì, io solitamente la metto in questi termini: ad ogni monoide M si associa una categoria BM, ad ogni categoria con un solo oggetto si associa un monoide i cui elementi sono gli endomorfismi di questo oggetto... e queste due corrispondenze sono mutuamente inverse.

Hai ragione che "sono precisamente" è impreciso perché dopo un po' questa equivalenza diventa parte del DBA di ognuno :D per inciso, questo è esattamente il puntiglio che vorrei l'esposizione avesse (per il bene dei lettori).